### ISSUE TITLE
Export annotations in Napari-compatible format

### ISSUE THREAD

**Issue by hannahker** [mlexchange/mlex_highres_segmentation#28]
Based on outputs of #25 

Annotations should be associated with a given image slice. So when multiple slices have annotations, the output should be 3-dimensional, rather than flattened into 2D. 

Also include in the output:
- What is the label for each class? (`{1: "box"}`) - these are user-defined names (depends on #36)
- What colour is assigned to each class? - again, user defined (depends on #36)
- For the future: brightness and contrast settings for the labelling (depends on #38)

**Comment by hannahker** [comment]
Also include:
- What is the label for each class? (`{1: "box"}`) - these are user-defined names
- What colour is assigned to each class? - again, user defined
- Nice to have: brightness and contrast settings for the labelling

**Comment by hannahker** [comment]
Notes on full vs sparse array? For just the annotations, it is likely that we can just export a sparse array **across slices** - data isn't required where there is no labelling. Necessary information is about which slice number is labelled. This is also input into the ML model.
